### ISSUE TITLE
NTR: alpha-ketoglutarate high in blood (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8634]
New term request alpha-ketoglutarate high in blood (segal_071107224417): 2-ketoglutarate; 2-oxoglutarate  UMLS: 4229114

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000435
- label: Elevated circulating alpha-ketoglutarate concentration
- parents: [Abnormal circulating carboxylic acid concentration (HP:0004354)](https://hpo.jax.org/app/browse/term/HP:0004354)
- pmid: 
- def: The concentration of alpha-ketoglutarate in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8634
- synonyms: Alpha-ketoglutarate high in blood(NONE)
